site/gdocs/pages/Homepage.tsx

```diff
@@ -92,32 +74,31 @@ const AllTopicsSection = () => {
             <p className="body-2-regular span-cols-12">
                 All our data, research, and writing — topic by topic.
             </p>
-            {flattenedCategories.map((category) => (
+            {flattenedAreas.map((area) => (
                 <section
-                    key={category.name}
+                    key={area.name}
                     className="homepage-topic span-cols-12"
                 >
                     <h2 className="homepage-topic__topic-name h3-bold">
-                        {category.name}
+                        {area.name}
                     </h2>
                     <ul className="homepage-topic__topic-list display-3-regular">
-                        {category.entries.map(
-                            ({ slug, title, isSubcategoryHeading }) =>
-                                isSubcategoryHeading ? (
-                                    <li
-                                        className="homepage-topic__subtopic"
-                                        key={`subtopic-entry-${slug}`}
-                                    >
-                                        {title}:
-                                    </li>
-                                ) : (
-                                    <li
-                                        className="homepage-topic__topic-entry"
-                                        key={`topic-entry-${slug}`}
-                                    >
-                                        <a href={`/${slug}`}>{title}</a>
-                                    </li>
-                                )
+                        {area.children.map(({ slug, name }) =>
+                            slug ? (
+                                <li
+                                    className="homepage-topic__topic-entry"
+                                    key={`topic-entry-${slug}`}
+                                >
+                                    <a href={`/${slug}`}>{name}</a>
+                                </li>
+                            ) : (
+                                <li
+                                    className="homepage-topic__subtopic"
+                                    key={`subarea-${name}`}
+                                >
+                                    {name}
```

Comment:
not sure about the design here but we had a ":" after subarea names.

Incidentally, subareas don't show on staging, but ok on dev (some rebaking/caching shenanigans?)

Reply:
Yeah the tag graph on staging doesn't have the latest structure that prod does. It's a pain to re-import, but if I tore down and recreated this staging env it would use the prod data which is correct 👍